kube-monitoring/charts/templates/_helper.tpl

```diff
@@ -97,7 +97,7 @@ app.kubernetes.io/managed-by: {{ $root.Release.Service }}
 release: {{ $root.Release.Name | quote }}
 {{- end }}
 
-{{- define "kube-monitoring.dashboardSelectorLabels" }}
+{{- define "kubeMonitoring.dashboardSelectorLabels" }}
```

Comment:
```suggestion
{{- define "kubeMonitoring.dashboardSelectorAnnotations" }}
```

Reply:
Again, still labels. I just renamed the helper template.